rpc-client-api/Cargo.toml

```diff
@@ -0,0 +1,31 @@
+[package]
+name = "solana-rpc-client-api"
+version = "1.12.0"
+description = "Solana Client Common Utilities"
+authors = ["Solana Maintainers <maintainers@solana.foundation>"]
+repository = "https://github.com/solana-labs/solana"
+license = "Apache-2.0"
+homepage = "https://solana.com/"
+documentation = "https://docs.rs/solana-rpc-client-api"
+edition = "2021"
+
+[dependencies]
+base64 = "0.13.0"
+bs58 = "0.4.0"
+jsonrpc-core = "18.0.0"
+reqwest = { version = "0.11.11", default-features = false, features = ["blocking", "brotli", "deflate", "gzip", "rustls-tls", "json"] }
```

Comment:
This crate doesn't need these reqwest features right? Like the point is it only uses `reqwest::Error`

Reply:
Good point, thanks! I haven't yet made it back to winnow these

Reply:
I think I'm going to leave these alone for now. My intent is to follow with rework of this error type so that the `reqwest` dependency can be removed from this crate altogether.